share/functions/__fish_shared_key_bindings.fish

```diff
@@ -0,0 +1,75 @@
+function __fish_shared_key_bindings -d "Bindings shared between emacs and vi mode"
+    # These are some bindings that are supposed to be shared between vi mode and default/emacs mode
+    # They are supposed to be unrelated to text-editing
+    # This takes $argv so the vi-bindings can pass the mode they are valid in
```

Comment:
Yes, I realize my prior comment is pedantic. But one of the things about the fish source code that is annoying is the weird mixing of sentences and sentence fragments.